src/zeit/content/volume/volume.py

```diff
@@ -226,7 +226,12 @@ def change_contents_access(
             exclude_performing_articles=True, dry_run=False):
         constraints = [{'term': {'payload.document.access': access_from}}]
         if exclude_performing_articles:
-            to_filter = _find_performing_articles_via_webtrekk(self)
+            try:
+                to_filter = _find_performing_articles_via_webtrekk(self)
+            except Exception:
+                log.error("Error while retrieving data from webtrekk api",
+                          exc_info=True)
+                return []
```

Comment:
Öh. Moment. _Das_ bedeutet ja aber, dass er so tut, als gäbe es keine performing articles? Ich dachte wir wollten, dass er dann abbricht? also schon irgendwas `raise`, aber eben konkret

Reply:
In dem Fall macht er dann einfach nix und hat von keinem Content den access-Wert verändert. Er failt, jetzt (sehr) leise. Man kann auch rerraisen, wenn man will.

Reply:
ah. okay!